Packages/com.unity.inputsystem/InputSystem/Events/IMECompositionEvent.cs

```diff
@@ -89,50 +123,53 @@ public void Dispose()
             object IEnumerator.Current => Current;
         }
 
-        public int Count => size;
+        int m_Length;
+        /// <summary>
+        /// The number of characters in the current IME Composition
```

Comment:
Punctuation. Sentences in API docs should be properly formed.

Reply:
Added